uploadNeuroDB/NeuroDB/ExitCodes.pm

```diff
@@ -189,4 +189,4 @@ our $NO_VALID_MINC_CREATED = 190; # if no valid MINC file was created
 
 our $REGISTER_PROGRAM_FAILURE = 200; # if MNI::Spawn::RegisterPrograms failed
 
-## -- FROM
\ No newline at end of file
+## -- FROM
```

Comment:
should probably remove that `## -- FROM` comment from the last line of the ExitCodes.pm library